machine/reference/mount.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Mount directory from machine
+keywords: machine, mount, subcommand
+title: docker-machine mount
+---
+
+Mount directories from a machine to your local host, using `sshfs`.
+
+The notation is `machinename:/path/to/dir` for the argument; you can also supply an alternative mount point (default is the same dir path).
+
+## Example
+
+Consider the following example:
+
+```none
+$ mkdir foo
+$ docker-machine ssh dev mkdir foo
+$ docker-machine mount dev:/home/docker/foo foo
+$ touch foo/bar
+$ docker-machine ssh dev ls foo
+bar
+```
+
+
+Now you can use the directory on the machine, for mounting into containers.
+Any changes done in the local directory, will be reflected in the machine too.
+
+```none
+$ docker run -v /home/docker/foo:/tmp/foo busybox ls /tmp/foo
```

Comment:
where are we when running this command? Perhaps prepend this block with: 
`docker-machine ssh dev`